spec/integration/manifest.spec.yml

```diff
@@ -245,6 +245,14 @@ spec:
                   - organization
                   - division
                   - team
+    required_vars:
+      description: Required conditional variables for the package.
+      type: array
```

Comment:
Should this array require at least two items? What would be the use of this list with a single list of variables?